commons-db/src/main/java/com/palantir/nexus/db/DBType.java

```diff
@@ -30,7 +30,9 @@ public enum DBType {
     // AJ: the oracle jdbc url is missing a final "paren" - processing in DBMgr will fix this...
     ORACLE("oracle.jdbc.driver.OracleDriver", "SELECT 1 FROM dual", true),
     POSTGRESQL("org.postgresql.Driver", "SELECT 1", true),
-    H2_MEMORY("org.h2.Driver", "SELECT 1", true);
+    H2_MEMORY("org.h2.Driver", "SELECT 1", true),
+    // Sqlite is _only_ used for tests. Please see {@link SqliteOracleAdapter}.
+    SQLITE("org.sqlite.JDBC", "SELECT 1", false);
```

Comment:
Flagging this - needed because BasicSql determines the sqlite type via the jdbc url, which is sqlite and not oracle